lib/src/main/java/io/ably/lib/rest/AblyRest.java

```diff
@@ -140,4 +140,11 @@ public void statsAsync(Param[] params, Callback<AsyncPaginatedResult<Stats>> cal
 	public void onAuthUpdated(String token) throws AblyException {
 		/* Default is to do nothing. Overridden by subclass. */
 	}
+
+	/**
+	 * Authentication error occurred
+	 */
+	public void onAuthError(ErrorInfo errorInfo) {
```

Comment:
As above, can we not remove this completely from this object and hide in a sub-object so that we are only polluting the REST object with one attribute / object?